CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -439,7 +510,8 @@ def checkRowForMismatch(row, columnDict, fixedColumnDict, baseColumnIdx=None, ig
         mismatchCell.value = "Y"
         mismatchCell.style = curMismatchFillStyle
     else:
-        mismatchCell.value = "N"
+        if(mismatchCell.value!="Y"):
```

Comment:
bracket not need. Also please use pep-8 styling. There's no spaces after commas and general formatting issues.

Reply:
updated.